tools/freebayes/freebayes.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="freebayes" name="FreeBayes" version="1.0.2.29">
+<tool id="freebayes" name="FreeBayes" version="1.0.2.29_galaxy1">
```

Comment:
General question what do people think about `_1`, `_2` or `--1` and `--2` similar to conda build numbers?
I don't like this galaxy on, especially not if we want to have here a Galaxy independent tool description.